wake/actions/cart/updateItemQuantity.ts

```diff
@@ -65,18 +65,28 @@ const action = async (
     throw new HttpError(400, "Missing cart cookie");
   }
 
-  let data = await removeFromCart(props, cartId, ctx, headers);
+  const cart = await ctx.invoke.wake.loaders.cart(props, req)
+  const item = cart.products?.find(item => item?.productVariantId === props.productVariantId)
+  const quantityItem = item?.quantity ?? 0
+  const quantity = props.quantity - quantityItem
```

Comment:
is this the left quantity? can you explain this calculation ? (add a js doc on top of it)

Reply:
yes. I add the doc, can you check?